scripts/dispatch-dbt-workflow.sh

```diff
@@ -28,12 +38,22 @@ GH_TOKEN=$(curl -s -L \
     "$GH_TOKENS_URL" \
     | jq -r '.token')
 
-# Use the token to call the API and dispatch the workflow
-echo "Dispatching workflow"
-curl -s -L \
-    -X POST \
-    -H "Accept: application/vnd.github+json" \
-    -H "Authorization: Bearer ${GH_TOKEN}" \
-    -H "X-GitHub-Api-Version: 2022-11-28" \
-    "$GH_API_REPO"/actions/workflows/"$GH_DBT_WORKFLOW"/dispatches \
-    -d '{"ref": "master"}'
+# Use the token to call the API and dispatch the workflows
+echo "Dispatching workflows"
+for i in ${!GH_API_REPOS[*]}; do
+    GH_API_PARAMS=$(
+        if [ -z "${GH_API_WORKFLOW_INPUTS[$i]}" ]; then
+            echo '{"ref": "master"}'
+        else
+            echo "{\"ref\": \"master\", \"inputs\": {\"models\": \"${GH_API_WORKFLOW_INPUTS[$i]}\"}"
```

Comment:
**[Nitpick]** Hm, I might be reading wrong here but I think we may be missing a closing brace:

```suggestion
            echo "{\"ref\": \"master\", \"inputs\": {\"models\": \"${GH_API_WORKFLOW_INPUTS[$i]}\"}}"
```